int64 a
---
std_msgs/String path
